test/list.test.js

```diff
@@ -38,6 +38,14 @@ function PhoneCtor(label, num) {
   this.num = num;
 }
 
+describe('Does not break default Array functionality', function() {
+  it('allows creating an empty length with a specified length', function() {
+    const list = new List(4);
+    list.should.be.an.instanceOf(Array);
+    list.length.should.be.eql(4);
+  });
```

Comment:
Please verify what is the `itemType` and also what are the individual list items set to. For the latter, maybe you can use `list.toJSON().should.eql([undefined, undefined, undefined, undefined])`?